src/vs/workbench/contrib/chat/browser/media/chat.css

```diff
@@ -2416,6 +2416,17 @@ display: none;
 	padding: 2px 0;
 }
 
+.chat-todo-list-widget .todo-item[title] {
+	cursor: help;
```

Comment:
Should this be a question mark? Could we instead use the default (hand) icon to keep it consistent with the other hover icons?